narwhals/expr.py

```diff
@@ -433,6 +433,43 @@ def std(self, *, ddof: int = 1) -> Self:
         """
         return self.__class__(lambda plx: self._call(plx).std(ddof=ddof))
 
+    def skew(self) -> Self:
```

Comment:
If you fancy going one step further, [polars signature](https://docs.pola.rs/api/python/stable/reference/expressions/api/polars.Expr.skew.html#polars.Expr.skew) has a `bias` argument - I am also happy to keep it as a follow up if that's too much for one PR

Reply:
Interesting, will check it out. Any suggestions on how to easily expose this `bias` argument for Narwhals as well?

Reply:
A (maybe) sophisticated example of how such remapping would work is [`value_counts`](https://github.com/narwhals-dev/narwhals/blob/main/narwhals/series.py#L1990) - which however exists only for series, but you should get the gist of it

Reply:
I think we should leave this for a separate PR as I don't see a straightforward way to add this. I've matched all frameworks to the Polars behaviour (i.e. Biases Population Skewness).